api/ccache.go

```diff
@@ -62,15 +63,29 @@ func (s *Server) ccacheDelete(ctx *middleware.Context, req models.CCacheDelete)
 		}
 
 		if len(req.Expr) > 0 {
-			nodes, err := s.MetricIndex.FindByTag(req.OrgId, req.Expr, 0)
+			var parsingError bool
```

Comment:
why do we need this variable? can't we just check err for nil?

Reply:
thx for pushing your commits. i consider this comment answered since you already fixed it